### PR TITLE
Remove mq collisions

### PR DESCRIPTION
**Pull request by BrendanGalloway**

### CODE REVIEW

File: .github/workflows/continuous.yaml [hunk 1]
```diff
@@ -30,26 +30,20 @@ jobs:
...
+        uses: docker/setup-buildx-action@v3
       - id: auth
         name: Authenticate to Google Cloud
-        uses: google-github-actions/auth@v1
+        uses: google-github-actions/auth@v2
         with:
           token_format: 'access_token'
           workload_identity_provider: 'projects/${{ secrets.DEV_GKE_PROJECT_ID}}/locations/global/workloadIdentityPools/github/providers/github'
           service_account: '${{ secrets.DEV_GKE_SA }}'
-      - name: Login to GCR
```
Comment (edamboritz) [inline .github/workflows/continuous.yaml:45]: I take it this step is no longer needed (and in other removed places)?

File: .github/workflows/helm.yaml [hunk 2]
```diff
@@ -43,8 +43,6 @@ jobs:
         with:
           # Fetch entire history. Required for chart-releaser; see https://github.com/helm/chart-releaser-action/issues/13#issuecomment-602063896
           fetch-depth: 0
-      - name: fetch head of branch to ensure semantic can run
```
Comment (edamboritz) [inline .github/workflows/helm.yaml:46]: Is this step also not needed

File: .github/workflows/continuous.yaml (hunk 1 above)
Comment (BrendanGalloway) [reply .github/workflows/continuous.yaml:45]: yes, it's a login to the old container registry which we've replaced with the artifact registry - we removed the push to this registry some time ago, I see the login was not taken out at the same time

File: .github/workflows/helm.yaml (hunk 2 above)
Comment (BrendanGalloway) [reply .github/workflows/helm.yaml:46]: hmm, this should not have been part of this change, will revert

**Review by edamboritz — APPROVED** [review]

**Pull request merged**